From 21af8f6d21b28a53041affb11b58d6316023fa92 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Wed, 26 Sep 2012 20:45:10 +0200
Subject: [PATCH 14/21] translations: fix phony translation linking error

  | .../usr/lib/crt1.o: In function `_start':
  | .../../sysdeps/i386/elf/start.S:115: undefined reference to `main'
  | collect2: ld returned 1 exit status

Upstream-Status: Pending

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 translations/translations.pro | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/translations/translations.pro b/translations/translations.pro
index dc99beb..0d98829 100644
--- a/translations/translations.pro
+++ b/translations/translations.pro
@@ -101,7 +101,7 @@ updateqm.name = LRELEASE ${QMAKE_FILE_IN}
 updateqm.CONFIG += no_link
 QMAKE_EXTRA_COMPILERS += updateqm
 
-isEmpty(vcproj) {
+!isEmpty(vcproj) {
     QMAKE_LINK = @: IGNORE THIS LINE
     OBJECTS_DIR =
     win32:CONFIG -= embed_manifest_exe
@@ -111,7 +111,7 @@ isEmpty(vcproj) {
     phony_src.input = PHONY_DEPS
     phony_src.output = phony.c
     phony_src.variable_out = GENERATED_SOURCES
-    phony_src.commands = echo int main() { return 0; } > phony.c
+    phony_src.commands = echo \"int main() { return 0; }\" > phony.c
     phony_src.name = CREATE phony.c
     phony_src.CONFIG += combine
     QMAKE_EXTRA_COMPILERS += phony_src
-- 
1.8.0

